src/Admin/UserRESTEndpointExtension.php

```diff
@@ -57,7 +57,13 @@ public function register_rest_field() {
 				'update_callback' => [ $this, 'update_review_panel_dismissed_for_template_mode' ],
 				'schema'          => [
 					'description' => __( 'For which template mode the Review panel on the Settings screen was dismissed by a user', 'amp' ),
-					'type'        => [ 'string', 'bool' ],
+					'type'        => 'string',
+					'enum'        => [
+						null,
```

Comment:
Is `null` still valid here? Given the `type` is `string`.

Reply:
Aside: I don't believe that `null` can be stored in in user meta. Storing `null` results in an empty string being returned. Similarly, the `get_review_panel_dismissed_for_template_mode` method says it returns a `string` or `WP_Error`, not `null`.

Reply:
So should this rather be:

```suggestion
						'',
```

Reply:
This was a bit of a puzzle to me. Another approach I considered was to set

```php
'type' => [ 'string', 'null' ]
```

but then the field validation didn't recognize `null` as an allowed `enum` value.

> I don't believe that null can be stored in in user meta.

What we're actually doing if the value is `empty` is deleting the user meta.

Anyway, let's go with an empty string (which we eventually received in `update_review_panel_dismissed_for_template_mode` even if `null` was used here).